[2024-11-09T14:02:38.061+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-09T14:02:38.087+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-09T14:02:30.232627+00:00 [queued]>
[2024-11-09T14:02:38.095+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-09T14:02:30.232627+00:00 [queued]>
[2024-11-09T14:02:38.097+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-09T14:02:38.108+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_json> on 2024-11-09 14:02:30.232627+00:00
[2024-11-09T14:02:38.112+0000] {standard_task_runner.py:64} INFO - Started process 243 to run task
[2024-11-09T14:02:38.114+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_json', 'manual__2024-11-09T14:02:30.232627+00:00', '--job-id', '314', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmp_515sglp']
[2024-11-09T14:02:38.116+0000] {standard_task_runner.py:91} INFO - Job 314: Subtask fetch_data_from_json
[2024-11-09T14:02:38.156+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_json manual__2024-11-09T14:02:30.232627+00:00 [running]> on host a5b668d467ee
[2024-11-09T14:02:38.259+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_json' AIRFLOW_CTX_EXECUTION_DATE='2024-11-09T14:02:30.232627+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-09T14:02:30.232627+00:00'
[2024-11-09T14:02:38.261+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-09T14:02:38.262+0000] {logging_mixin.py:188} INFO - Error reading JSON file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.json'
[2024-11-09T14:02:38.262+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-09T14:02:38.263+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-09T14:02:38.272+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_json, run_id=manual__2024-11-09T14:02:30.232627+00:00, execution_date=20241109T140230, start_date=20241109T140238, end_date=20241109T140238
[2024-11-09T14:02:38.292+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-09T14:02:38.314+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-09T14:02:38.319+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
